new-package/elm-review-package-tests/helpers/find-configurations.js

```diff
@@ -10,11 +10,17 @@ module.exports = {
   findPreviewConfigurations
 };
 
+/**
+ * @returns {string[]}
+ */
 function findPreviewConfigurations() {
-  return glob
-    .sync(`${root}/preview*/**/elm.json`, {
-      ignore: ['**/elm-stuff/**'],
-      nodir: true
-    })
-    .map(path.dirname);
+  return (
+    glob
+      .sync(`${root}/preview*/**/elm.json`, {
+        ignore: ['**/elm-stuff/**'],
+        nodir: true
+      })
+      // eslint-disable-next-line @typescript-eslint/unbound-method -- I don't know how to fix this.
```

Comment:
This will make it into the users' project. I'd rather have the rule disabled for this file (by defining an exception in the ESLint config, not a different disable comment at the start of the file) than to have a disable comment.

Reply:
Fair. I'll switch it when I get a few minutes.